### PR TITLE
Enable quantum resistant tunnels by default on Windows

### PR DESCRIPTION
**Pull request by dlon**
Close DES-559.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6804)

### CODE REVIEW

**Review by MarkusPettersson98 — DISMISSED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6804#-:-O7DiLa93gyyxfq1CG1O:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ Line wrap the file at 100 chars.                                              Th
   multihop, quantum-resistant tunnels, or DAITA.
 - Improved output format of `mullvad status` command, which now also prints feature indicators.
 
+#### Windows
+- Enable quantum resistant tunnels by default (when set to `auto`).
```
Comment (MrChocolatine) [inline CHANGELOG.md:39]: Just a friendly note, I see both "quantum resistant" and "quantum-resistant" used,
but perhaps there is one format you prefer (internally at Mullvad). :v:

Comment (dlon) [reply CHANGELOG.md]: Thanks, I've made the changelog consistent.

**Review by MarkusPettersson98 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/6804#-:-O7ZTsPKC9CcPGIeWUbM:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @MrChocolatine)

**Pull request merged**